src/lib/utils.js

```diff
@@ -171,15 +171,40 @@ const argsToCamelCase = (object) => {
     return camelCasedObject;
 };
 
+const createSourceFiles = async (paths) => {
+    return paths.map((filePath) => {
+        const file = fs.readFileSync(filePath);
+        const contentType = mime.getType(filePath) || 'text/plain';
+        const format = (contentType.startsWith('text/') || contentType.includes('javascript') || contentType.includes('json'))
```

Comment:
This is not great as it doesn't cover for example `application/xml` but I am not able to find any good source of text mime types. If we don't resolve this then add some TODO here..